task/gitleaks/0.1/samples/run.yaml

```diff
@@ -0,0 +1,55 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: gitleaks-pipeline
+spec:
+  workspaces:
+  - name: workspace
+  tasks:
+  - name: clone-git-repo
+    workspaces:
+    - name: output
+      workspace: workspace
+    taskRef:
+      name: git-clone
```

Comment:
Should we use bundle here ??

Reply:
cc @afrittoli  @vdemeester  @piyush-garg

Reply:
Hey @PuneetPunamiya. I don't have much idea about bundle. Could you please elaborate why it should be used here?

Reply:
Using Tekton Bundle we can refer the task by creating a bundle of that task and pushing it to some OCI registry for e.g.

```
apiVersion: tekton.dev/v1beta1
kind: Pipeline
metadata:
  name: <pipeline-name>
spec:
  taskRef:
    name: <task-name>
    bundle: <image-name>   ---> Registery to which image for Task is pushed
```

But I think I'm wrong and we don't need this here

Reply:
Okay Thanks 😄